playbooks/archivematica-src-centos-7/Vagrantfile

```diff
@@ -0,0 +1,47 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  config.vm.box = ENV.fetch("VAGRANT_BOX", "centos/7")
+
+  {
+    "am-local-centos7" => {
+      "ip" => "192.168.168.197",
+      "memory" => "3072",
```

Comment:
This is more than the archivematica (source) installation - any particular reason?